docs/wallets/app-wallet/transfer-hotspot.mdx

```diff
@@ -24,7 +24,7 @@ transferred, the new owner will receive all future Hotspot mining rewards.
 
 - This feature requires both users to have a valid Helium wallet account
 - The Seller pays the transaction fee of ~55,000 DC.
-- Hotspots must have had Proof-of-Coverage activity (beaconing or witnessing) in the last 1200
+- Hotspots must have had Proof-of-Coverage activity (beaconing or witnessing) or data transfer activity in the last 1200
```

Comment:
```suggestion
- Hotspots must have had Proof-of-Coverage activity (beaconing or witnessing) or Data Transfer
  activity in the last 1200 blocks (approximately 20 hours) in order to be transferred. This is to
  ensure the Hotspot is fully functional for the protection of the buyer.
```

small formatting fix, but otherwise this is all good, thank you!